src/main/java/org/usfirst/frc4904/robot/commands/VisionAngleAlign.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisIdle;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisTurn;
+
+/**
+ * Rotate the robot in the direction of the vision target
+ */
+public class VisionAngleAlign extends ChassisTurn {
```

Comment:
Rename to make it clear that this turns to a shooting angle, without moving into a box shot.